packages/react-native/src/Authenticator/hooks/useRouteTypedFields/utils.ts

```diff
@@ -54,7 +54,7 @@ export const getTypedField = ({
       }
 
       if (isPhoneFieldType && key === 'dialCodeList') {
-        return { ...acc, dialCodes: value as string[] };
+        return { ...acc, dialCodes: undefined };
```

Comment:
While this is helpful for the default `fields`, this won't prevent the picker from rendering when users pass in custom fields with a `dialCodes` prop

Reply:
Right but this is a temporary change, as we will add dialcodes before dev preview.

Reply:
I'm out of context here, is `dialCodes` not implemented yet?

Reply:
@ioanabrooks Regardless of what happens in the future this will not prevent users currently testing the component from rendering the Picker in a `PhoneNumberField`. Please see: https://github.com/aws-amplify/amplify-ui/commit/7e544f77d98fd1643dbac76f07c0f9129fe4ee09